mw_api_rate_limit_test.go

```diff
@@ -46,13 +47,14 @@ func getGlobalRLAuthKeyChain(spec *APISpec) http.Handler {
 	proxy := TykNewSingleHostReverseProxy(remote, spec)
 	proxyHandler := ProxyHandler(proxy, spec)
 	baseMid := BaseMiddleware{spec, proxy}
+	rlMW := &RateLimitForAPI{BaseMiddleware: baseMid}
 	chain := alice.New(mwList(
 		&IPWhiteListMiddleware{baseMid},
 		&AuthKey{baseMid},
 		&VersionCheck{BaseMiddleware: baseMid},
 		&KeyExpired{baseMid},
 		&AccessRightsCheck{baseMid},
-		&RateLimitForAPI{BaseMiddleware: baseMid},
+		rlMW,
```

Comment:
Why the vars?

Reply:
Ah, I thought the init wasn't getting called, so I was calling them manually.